.github/workflows/bbw_build_container.yml

```diff
@@ -100,6 +111,12 @@ jobs:
             image: opensuse/leap:15.3
             tag: opensuse15
             platforms: linux/amd64
+          - dockerfile: centos.Dockerfile
+            image: rockylinux:8
```

Comment:
Same as AlmaLinux - just an install test from the RHEL binaries in a VM is sufficient.